### PR TITLE
Miscellaneous cleanups

### PR DESCRIPTION
**Pull request by bbannier**

### CODE REVIEW

File: tests/Baseline/ztest.01_success_not_suppressed/output [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
```
Comment (jlucovsky) [inline tests/Baseline/ztest.01_success_not_suppressed/output:1]: Why do all the baselines contain this comment -- BTest -- for a ztest cleanup?

File: tests/Scripts/canonify-ztest [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+# Get us "modern" regexps with sed.
+if [ `uname` == "Linux" ]; then
+   sed="sed -r"
+else
```
Comment (jlucovsky) [inline tests/Scripts/canonify-ztest:8]: Suggest adding a comment for this case ... if it's not `Linux` is is MacOS/Darwin or Windows? Please clarify the system expected. On Windows, is a cygwin environment or the windows linux subsystem required?

File: tests/btest.cfg [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[btest]
+TestDirs    = ztest
+TmpDir      = %(testbase)s/.tmp
+BaselineDir = %(testbase)s/Baseline
+IgnoreDirs  = .svn CVS .tmp
```
Comment (jlucovsky) [inline tests/btest.cfg:5]: add `.git`?

**Review by jlucovsky — COMMENTED** [review]
Nit-picky feedback

File: tests/btest.cfg (hunk 3 above)
Comment (bbannier) [reply tests/btest.cfg:5]: This is the default BTest config. The reason it probably doesn't add `.git` is that that folder can only be found in the root of the project and not e.g., in this subdirectory.

File: tests/Baseline/ztest.01_success_not_suppressed/output (hunk 1 above)
Comment (bbannier) [reply tests/Baseline/ztest.01_success_not_suppressed/output:1]: I added a patch which adds BTest baselines which ensure that the ZTest output is of an expected format. BTest insert this comment for automatically created baselines.

File: tests/Scripts/canonify-ztest (hunk 2 above)
Comment (bbannier) [reply tests/Scripts/canonify-ztest:8]: Done.

**Review by awelzel — APPROVED** [review]
I've skimmed through the changes, fwiw.

**Comment by J0eJ0h** [comment]
> I've skimmed through the changes, fwiw.

Same here.

**Pull request merged**